### PR TITLE
feat(cdr): Docs for remote postgresql C4D

### PR DESCRIPTION
**Pull request by Emyrk**
@khorne3 we should wait until 1.28 to put this in. It _works_ in 1.27, but has some quirks I fixed.

### CODE REVIEW

File: setup/docker.md [hunk 1]
```diff
@@ -91,6 +91,45 @@ For example:
 docker run --rm -it -p 7080:7080 -v /var/run/docker.sock:/var/run/docker.sock -v ~/.coder:/var/run/coder -e DEVURL_HOST="*.mycompany.com" codercom/coder:1.27.0
 ```
 
+## External PostgreSQL database
+
+If you'd like to use an external database, you must:
+
+1. Disable the embedded database with the environment variable `DB_EMBEDDED`
+1. pass in the database connection information to the external PostgreSQL
```
Comment (jawnsy) [inline setup/docker.md:99]: I think Pass should be capitalized here, and changed to something like "Provide"

**Review by jawnsy — APPROVED** [review]
Looking good

**Review by khorne3 — APPROVED** [review]

File: setup/docker.md [hunk 2]
```diff
@@ -91,6 +91,45 @@ For example:
...
+   database:
+
+```bash
+docker run --rm -it -p 7080:7080 \
```
[the remark this answers is not in the document]
Comment (jawnsy) [reply setup/docker.md]: it does add a dependency, but I think examples should be exemplary - that is, examples should be the "recommended" approach that people should use. if we're specifying things as you do here, then what you're implicitly saying is that we recommend people manage Coder this way, and I don't think that's the case...

**Comment by khorne3** [comment]
merging into release branch, but am open to updates as needed/wanted

**Pull request merged**